packages/next-swc/crates/napi/src/transform.rs

```diff
@@ -71,90 +71,99 @@ impl Task for TransformTask {
     type JsValue = Object;
 
     fn compute(&mut self) -> napi::Result<Self::Output> {
-        GLOBALS.set(&Default::default(), || {
-            let eliminated_packages: Rc<RefCell<fxhash::FxHashSet<String>>> = Default::default();
-            let res = catch_unwind(AssertUnwindSafe(|| {
-                try_with_handler(
-                    self.c.cm.clone(),
-                    turbopack_binding::swc::core::base::HandlerOpts {
-                        color: ColorConfig::Always,
-                        skip_filename: skip_filename(),
-                    },
-                    |handler| {
-                        self.c.run(|| {
-                            let options: TransformOptions = serde_json::from_slice(&self.options)?;
-                            let fm = match &self.input {
-                                Input::Source { src } => {
-                                    let filename = if options.swc.filename.is_empty() {
-                                        FileName::Anon
-                                    } else {
-                                        FileName::Real(options.swc.filename.clone().into())
+        tokio::runtime::Runtime::new()
```

Comment:
napi should already have a tokio runtime, can we use it here somehow?
https://napi.rs/docs/compat-mode/concepts/tokio

Reply:
We can't use it for `AsyncTask`